v0/destinations/posthog/config.js

```diff
@@ -0,0 +1,44 @@
+const { getMappingConfig } = require("../../util");
+
+const BASE_ENDPOINT = "https://app.posthog.com";
+
+const CONFIG_CATEGORIES = {
+  ALIAS: {
+    endPoint: BASE_ENDPOINT,
```

Comment:
I think we won't need the `endPoint` in the config as we'll be using the same endPoint for all kinds of events.

Reply:
Fixed It